src/utils.js

```diff
@@ -0,0 +1,7 @@
+export const getTranslationPath = (path, locale) => {
+  if (typeof path === 'function') {
+    return path(locale);
+  }
+
+  return `${path}${locale}.json`;
```

Comment:
Not sure if this will work, our translation file names for frontend apps are defined inside the `asset-manifest.json`, they are mapped by locale and the files get suffixed with a content-hash

Reply:
This is just the current behaviour. I haven't touched it. But in this case, it makes sense. Because not every microservice might make use of the manifest. And those probably do not need a function.

Reply:
👍